daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -85,3 +94,137 @@ func (l *JSONFileLogger) Close() error {
 func (l *JSONFileLogger) Name() string {
 	return Name
 }
+
+func decodeLogLine(dec *json.Decoder, l *jsonlog.JSONLog) (msg logger.Message, err error) {
```

Comment:
You definitely don't need named returns here. And use named literals for creating msg too pls